src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -1181,26 +1210,26 @@ object LazyList extends SeqFactory[LazyList] {
     case _                        => newLL(stateFromIterator(coll.iterator))
   }
 
-  def empty[A]: LazyList[A] = _empty
+  def empty[A]: LazyList[A] = Empty
 
   /** Creates a State from an Iterator, with another State appended after the Iterator
     * is empty.
     */
-  private def stateFromIteratorConcatSuffix[A](it: Iterator[A])(suffix: => State[A]): State[A] =
+  private def stateFromIteratorConcatSuffix[A](it: Iterator[A])(suffix: => LazyList[A]): LazyList[A] =
     if (it.hasNext) sCons(it.next(), newLL(stateFromIteratorConcatSuffix(it)(suffix)))
     else suffix
 
   /** Creates a State from an IterableOnce. */
-  private def stateFromIterator[A](it: Iterator[A]): State[A] =
+  private def stateFromIterator[A](it: Iterator[A]): LazyList[A] =
     if (it.hasNext) sCons(it.next(), newLL(stateFromIterator(it)))
-    else State.Empty
+    else LazyList.Empty
```

Comment:
I don't think any of these need to be prefixed as `LazyList.`

```suggestion
    else Empty
```